res/satis-schema.json

```diff
@@ -284,6 +284,10 @@
         "_comment": {
             "type": ["array", "string"],
             "description": "A key to store comments in"
+        },
+        "pretty-print": {
+            "type": "boolean",
+            "description": "Defaults to true. If true, the output files will NOT be minified (will look pretty tough)."
```

Comment:
I think this description is a bit misleading. Minification is not really the same as simply having no formatting whitespace.

Reply:
updated